service/grpc/codelingo/codelingo.pb.go

```diff
@@ -162,94 +162,79 @@ type PathsFromOffsetRequest struct {
 	End      int64  `protobuf:"varint,6,opt,name=end" json:"end,omitempty"`
 }
 
-func (m *PathsFromOffsetRequest) Reset()                    { *m = PathsFromOffsetRequest{} }
-func (m *PathsFromOffsetRequest) String() string            { return proto.CompactTextString(m) }
-func (*PathsFromOffsetRequest) ProtoMessage()               {}
-func (*PathsFromOffsetRequest) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{3} }
+func (m *QueryFromOffsetRequest) Reset()                    { *m = QueryFromOffsetRequest{} }
+func (m *QueryFromOffsetRequest) String() string            { return proto.CompactTextString(m) }
```

Comment:
exported method QueryFromOffsetRequest.ProtoMessage should have comment or be unexported